pkg/pss/pss_test.go

```diff
@@ -0,0 +1,223 @@
+// Copyright 2020 The Swarm Authors. All rights reserved.
+// Use of this source code is governed by a BSD-style
+// license that can be found in the LICENSE file.
+
+package pss_test
+
+import (
+	"context"
+	"reflect"
+	"testing"
+	"time"
+
+	"github.com/ethersphere/bee/pkg/pss"
+	"github.com/ethersphere/bee/pkg/storage"
+	"github.com/ethersphere/bee/pkg/storage/mock"
+	"github.com/ethersphere/bee/pkg/swarm"
+	"github.com/ethersphere/bee/pkg/tags"
+	"github.com/ethersphere/bee/pkg/trojan"
+)
+
+// TestTrojanChunkRetrieval creates a trojan chunk
+// mocks the localstore
+// calls pss.Send method and verifies it's properly stored
+func TestTrojanChunkRetrieval(t *testing.T) {
+	var err error
+	ctx := context.TODO()
+	testTags := tags.NewTags()
+
+	localStore := mock.NewTagsStorer(testTags)
+	pss := pss.NewPss(localStore, testTags)
+
+	target := trojan.Target([]byte{1}) // arbitrary test target
+	targets := trojan.Targets([]trojan.Target{target})
+	payload := []byte("RECOVERY CHUNK")
```

Comment:
perhaps choose a more neutral string as value. pss is not a recovery-specific package

Reply:
We were using this pss only for Global Pinning that's why it got hardcoded.
I will change it to a generic topic

Reply:
fixed